docs-src/typescript/clients.md

```diff
@@ -57,7 +57,7 @@ async function run() {
     // type inference works! args: [name: string]
     args: ['Temporal'],
     taskQueue: 'hello-world',
-    // in practice, use a meaningful business id, eg customerId or transactionId
+    // in practice, use a meaningful business ID, like customerId or transactionId
```

Comment:
```suggestion
    // in practice, use a meaningful business identifier, like customerId or transactionId
```
Per https://github.com/temporalio/documentation/blob/main/STYLE.md#abbreviation-of-identifier.